functions/Get-DbaTable.ps1

```diff
@@ -76,22 +76,43 @@ Returns information on the CommandLog table in the DBA database on both instance
 		
 		if ($Table) {
 			foreach ($t in $Table) {
-				$dotcount = ([regex]::Matches($t, "\.")).count
-				
+                $splitName = [regex]::Matches($t, "(\[.+?\])|([^\.]+)").Value
```

Comment:
As we talked, I have added some verbosity to the help for Table parameter and added new example.